Simulator/Vertices/Neuro/AllSpikingNeurons.cpp

```diff
@@ -19,6 +20,10 @@ void AllSpikingNeurons::setupVertices()
 
    hasFired_.assign(size_, false);
    vertexEvents_.assign(size_, maxSpikes);
+
+   // register variables
+   Recorder &recorder = *Simulator::getInstance().getModel()->getRecorder();
```

Comment:
OK, hate to do this, but I know that @KamathScience has transitioned a lot of the `get*()` methods to return references, rather than pointers. I don't know if that applies here, or if it has made its way into documentation (though that would almost certainly not be the `gh-pages` documentation, yes). Consult with her about this.